modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -1017,6 +1058,7 @@ def obj.class
                     allow_any_instance_of(MPI::Service).to receive(:find_profile).and_return(mvi_profile_response)
 
                     post path, params: data, headers: auth_header
+                    expect(response.status).to eq(200)
```

Comment:
Forgot to add this in to https://github.com/department-of-veterans-affairs/vets-api/pull/9642 . Basically just makes sure this is _actually succeeding_ . The test was throwing 422s and we were just missing it